### ISSUE TITLE
Maintainer

### ISSUE THREAD

**Issue by ketzacoatl** [beijaflor-io/haskell-language-hcl#9]
Would it help to have an additional maintainer for this project? I would be willing to lend a hand in getting the library updated and managing incoming PRs.

**Comment by yamadapc** [comment]
Yes, I don't have time / reason to maintain this library at the moment so I'd be happy to:

* Add you to the project
* Not sure if I can grant you access to Hackage, but that'd be fine by me

However, it might be a better suggestion that somebody else maintains a fork.

**Comment by ketzacoatl** [comment]
Hi @yamadapc, thank you for the reply.

What would be better in the forking scenario?

That would change the package name on hackage, or retain the same?

I'm not too strongly opposed or attached to a particular route - it's not really my original work, I'm just looking to carry it forward in some form :)

**Comment by yamadapc** [comment]
I’m not sure. If you have time I’ll just grant you access to the repository.

Fork would change the package name yes; but I’d be surprised if there’re a lot of consumers at this point as this library hasn’t been updated in years.

**Comment by ketzacoatl** [comment]
I don't have a lot of time, but I intend to chip away at things.

I'm fine creating `haskell-hcl` or `haskell-language-hcl2` as a fork, if you'd prefer to archive this or otherwise leave it be.
